runtime/proto/service/corfu_message.proto

```diff
@@ -93,6 +93,8 @@ message RequestPayloadMsg {
     LogReplicationEntryMsg lr_entry = 70;
     LogReplicationMetadataRequestMsg lr_metadata_request = 71;
     LogReplicationLeadershipRequestMsg lr_leadership_query = 72;
+    // sentinel value for the LogReplicationMsgHandler
+    LogReplicationEntryMsg none = 73;
```

Comment:
can you explain the usage of this request type? is it for the subscription?

Reply:
No, its not for subscription.
We now have 1 handler for both source/sink, as opposed to have them different like it was before.
The annotation [LogReplicationMsgHandler](https://github.com/CorfuDB/CorfuDB/blob/connectionModelRedesign/infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/msgHandlers/LogReplicationMsgHandler.java) has 2 methods, 1 for request payload type and 1 for response payload type.

In the handle* APIs in [LogReplicationServer](https://github.com/CorfuDB/CorfuDB/blob/connectionModelRedesign/infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/msgHandlers/LogReplicationServer.java#L198), we either need to provide value for both the methods, or have a default value and set the msg payload to the appropriate method.
So I have the "none" payload type in both the request-payload and response-payload which acts as a default.

Reply:
Discussed today, changing to having 2 annotations which will not need the sentinel value